packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -418,6 +418,11 @@ - (NSView *)nextViewToFocusOutsideZone:(FocusZoneAction)action
 		nextViewToFocus = [self nextValidKeyView];
 		while([nextViewToFocus isDescendantOf:focusZoneAncestor])
```

Comment:
Ahhh I didn't realize that `isDescendantOf:` returns `YES` for `self`. Good to know!

Reply:
we could add a check here `&& nextViewToFocus != self` ?

Reply:
@Saadnajmi That would make the code cleaner but I'd need to set `nextViewToFocus` to nil when `nextViewToFocus == self` returns true? Unless there's a workaround for that 😅

Reply:
I will say that I also relied on `isDescendantOf` returning self when I first added that logic... but I can't remember why. Simplified something